core/block/import/notion/api/block/link.go

```diff
@@ -230,14 +239,14 @@ func (b BookmarkObject) GetBookmarkBlock() (*model.Block, string) {
 // returned to us by the Notion API. As a result we get Anytype ID of child page and make it targetBlockID
 // But we can end up with 3 links to the same page, and to avoid that,
 // we remove the childID from the parentIDToChildrenID map. So it helps to not create links with the same targetBlockID.
-func getTargetBlock(parentPageIDToChildIDs map[string][]string, pageIDToName, notionIDsToAnytype map[string]string, pageID, title string) (string, error) {
+func getTargetBlock(importContext *NotionImportContext, pageIDToName, notionIDsToAnytype map[string]string, pageID, title, blockID string) (string, error) {
```

Comment:
blockID -> parentBlockID?

Reply:
also seems like it applies to all other occurrences in this pull request

Reply:
Fixed